src/workers/webdav/worker/InternxtFileSystem/InternxtFileSystem.ts

```diff
@@ -132,6 +137,9 @@ export class InternxtFileSystem extends FileSystem {
     }
 
     if (item.isFile()) {
+      const isValidName = this.fileValidator.validatePath(path);
```

Comment:
If we already have an instance of a Webdav we know that the file is correct, since the path is validated when its created